code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -242,7 +242,7 @@
 /datum/gear/accessory/detbadge
 	display_name = "badge, detective"
```

Comment:
Rename that to investigator, as its inconsistent otherwise.

Reply:
Done.